admin/src/components/filters-system-v2/components/filters/FilterPopOver.tsx

```diff
@@ -1,26 +1,40 @@
 import { Popover, Transition } from "@headlessui/react";
-import React, { Fragment } from "react";
+import React, { Fragment, ReactElement } from "react";
 import { BsChevronRight } from "react-icons/bs";
 import Trash from "../../../../assets/icons/Trash";
 import { normalizeString } from "./utils";
+import { RowFilter, IIntermediateFilter, DataFilter } from "@/components/filters-system-v2/components/Filter";
+import { IntermediateFilterCount, syncRootFilter } from "@/components/filters-system-v2/components/filters/IntermediateFilter";
 
 // file used to show the popover for the all the possible values of a filter
 
 function classNames(...classes) {
   return classes.filter(Boolean).join(" ");
 }
+type FilterPopOverProps = {
```

Comment:
```suggestion
interface Props = {
```
IMHO `Props` suffit ici car un seul composant dans le fichier et pas d'export, il n'est pas amené à être réutilisé

Reply:
Je préfère être explicite dans le nommage, mais en effet, ça se discute :-)